src/EnergyPlus/SetPointManager.cc

```diff
@@ -5341,7 +5358,24 @@ namespace SetPointManager {
 			this->MySetPointCheckFlag = false;
 		}
 
-		this->SetPt = Node( RefNode ).TempSetPoint - ( Node( FanOutNode ).Temp - Node( FanInNode ).Temp );
+		if ( CoolCoilInNode == 0 && CoolCoilOutNode == 0 ) {
+			this->SetPt = Node( RefNode ).TempSetPoint - ( Node( FanOutNode ).Temp - Node( FanInNode ).Temp );
```

Comment:
Mike:

I made code change slightly as follow:

		if ( CoolCoilInNode == 0 && CoolCoilOutNode == 0 ) {
			this->SetPt = Node( RefNode ).TempSetPoint - ( Node( FanOutNode ).Temp - Node( FanInNode ).Temp );
		} else {
			dtFan = Node( FanOutNode ).Temp - Node( FanInNode ).Temp;
			dtCoolCoil = Node( CoolCoilInNode ).Temp - Node( CoolCoilOutNode ).Temp;
			if ( dtCoolCoil > 0.0 && MinTemp > OutDryBulbTemp ) {
				this->FreezeCheckEnable = true;
			}
			if ( this->FreezeCheckEnable ) {
				if ( Node( RefNode ).Temp == Node( CoolCoilOutNode ).Temp ) { // blow through
					this->SetPt = max( Node( RefNode ).TempSetPoint, MinTemp ) - dtFan + dtCoolCoil;
				} else { // draw through
					if ( RefNode != CoolCoilOutNode ) { // Ref node is outlet node
						this->SetPt = max( Node( RefNode ).TempSetPoint - dtFan, MinTemp ) + dtCoolCoil;
					} else {
						this->SetPt = max( Node( RefNode ).TempSetPoint, MinTemp ) + dtCoolCoil;
					}
				}
			} else {
				this->SetPt = Node( RefNode ).TempSetPoint - ( Node( FanOutNode ).Temp - Node( FanInNode ).Temp );
			}
		}

When FreezeCheckEnable is false, the original calculation remains. However, differences of example runs are found.